tests/migration_test.go

```diff
@@ -4917,3 +4916,13 @@ func affinityToMigrateFromSourceToTargetAndBack(sourceNode *k8sv1.Node, targetNo
 		},
 	}, nil
 }
+
+func filterRunningMigrations(migrations []v1.VirtualMachineInstanceMigration) []v1.VirtualMachineInstanceMigration {
+	runningMigrations := []v1.VirtualMachineInstanceMigration{}
+	for _, migration := range migrations {
+		if migration.IsRunning() {
+			runningMigrations = append(runningMigrations, migration)
+		}
+	}
+	return runningMigrations
+}
```

Comment:
I guess you don't want to "trust" `migrations.FilterRunningMigrations()`, but I don't really see the point of duplicating it...
It's unit-tested, and we make sure the results are as expected after calling it.

Reply:
@jean-edouard The reason I've moved the function to e2e is because it receives slice of migration objects. It is common to use in e2e since we use direct API calls, in particular when we request a `List` we receive a slice of objects. OTOH in the product code we use informers, and when we request a list of objects from the cache store we receive slice of object pointers.